banzai/main.py

```diff
@@ -157,64 +157,64 @@ def process_single_frame(pipeline_context, raw_path, filename, last_stage=None,
         logger.error(e, extra_tags={'filename': filename})
 
 
-def parse_directory_args(pipeline_context, raw_path, selection_criteria, extra_console_arguments=None):
+def parse_directory_args(pipeline_context, raw_path, selection_criteria, image_class, extra_console_arguments=None):
     if extra_console_arguments is None:
         extra_console_arguments = []
 
     if pipeline_context is None:
         if raw_path is None:
             extra_console_arguments += [RAW_PATH_CONSOLE_ARGUMENT]
 
-        pipeline_context = parse_args(selection_criteria, extra_console_arguments=extra_console_arguments)
+        pipeline_context = parse_args(selection_criteria, image_class, extra_console_arguments=extra_console_arguments)
 
         if raw_path is None:
             raw_path = pipeline_context.raw_path
     return pipeline_context, raw_path
 
 
 def make_master_bias(pipeline_context=None, raw_path=None):
-    pipeline_context, raw_path = parse_directory_args(pipeline_context, raw_path, settings.IMAGING_CRITERIA)
+    pipeline_context, raw_path = parse_directory_args(pipeline_context, raw_path, settings.IMAGING_CRITERIA, Image)
```

Comment:
Maybe move `Image` to `settings.IMAGE_CLASS`?

Reply:
Done.